test/utils/test-swagger-gen.ts

```diff
@@ -265,6 +265,17 @@ const testSwaggers: { [name: string]: SwaggerConfig } = {
     addCredentials: false,
     isTestPackage: true
   },
+  customUrlRest: {

```

Comment:
Would you mind moving this to the bottom next to all the other REST Client tests?

Reply:
moved